scripts/init_db.py

```diff
@@ -0,0 +1,20 @@
+import os
+from sqlalchemy import create_engine
+from nad_ch.infrastructure.database import ModelBase
+from nad_ch.config import DATABASE_URL
+
+
+def main():
+    engine = create_engine(DATABASE_URL)
```

Comment:
Would be good to add an ADR for sqlalchemy usage

Reply:
Agree, adding one.